x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/Querier.java

```diff
@@ -201,7 +202,7 @@ public static void closePointInTime(Client client, BytesReference pointInTimeId,
     public static SearchRequest prepareRequest(SearchSourceBuilder source, SqlConfiguration cfg, boolean includeFrozen, String... indices) {
         source.timeout(cfg.requestTimeout());
 
-        SearchRequest searchRequest = new SearchRequest(INTRODUCING_UNSIGNED_LONG);
+        SearchRequest searchRequest = new SearchRequest(Version.fromId(INTRODUCING_UNSIGNED_LONG.id));
```

Comment:
This is the `minCompatibleShardNode` - which is still a `Version`. If the version is locked at 8.16, this means this parameter can be completely removed on v9 and replaced with another mechanism if/when SQL needs to be updated?

Reply:
OK - After this is merged, I'll look at removing that then